airflow/migrations/versions/0041_3_0_0_rename_dataset_as_asset.py

```diff
@@ -285,13 +285,6 @@ def upgrade():
             unique=False,
         )
 
-        batch_op.create_foreign_key(
-            constraint_name="toar_asset_fkey",
-            referent_table="asset",
-            local_cols=["asset_id"],
-            remote_cols=["id"],
-            ondelete="CASCADE",
-        )
```

Comment:
I moved this to a new file because, for some reason, SQLite could not create this FK in this file. I suspect it was because the asset table was renamed in this migration

Reply:
Only one remains now, and I have spent quality time trying to solve it but have yet to figure it out. Since this is sqlite only, should we merge and debug the issue in another PR?
Remaining one:
```
/usr/local/lib/python3.9/contextlib.py:126 SAWarning: WARNING: SQL-parsed foreign key constraint '('alias_id', 'asset_alias', 'id')' could not be located in PRAGMA foreign_keys for table asset_alias_asset
```